poms/bom/pom.xml

```diff
@@ -2185,7 +2190,7 @@
                 <artifactId>debezium-embedded</artifactId>
                 <version>${debezium.version}</version>
             </dependency>
-            <dependency>
+	    <dependency>
```

Comment:
```suggestion
            <dependency>
```

Reply:
Yeah I saw that. I'll address that in a follow up PR as I plan to address some of the outstanding TODO items.